src/ssh.rs

```diff
@@ -16,16 +15,17 @@ use std::time::{Duration, Instant};
 /// To execute a command and get its `STDOUT` output, use
 /// [`Session#cmd`](struct.Session.html#method.cmd).
 pub struct Session {
+    pub addr: SocketAddr,
     ssh: ssh2::Session,
     _stream: TcpStream,
 }
 
 impl Session {
-    pub(crate) fn connect(
+    pub fn connect(
```

Comment:
Why should this be `pub`?

Reply:
leaving aside tsunami, it's independently useful - making it public makes it possible to use in other crates.

alternately, this module could be its own crate?

Reply:
Like a "convenient SSH session" crate? That seems like a _great_ idea.

Reply:
https://crates.io/crates/sessh